packages/fast-components-react-base/src/hypertext/hypertext.props.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+/**
+ * Hypertext HTML tags
+ */
+export enum HypertextHTMLTags {
+    a = "a",
+}
+
+export interface IHypertextProps {
+
+    id?: string;
+
+    text?: string;
```

Comment:
Since we're passing children here, text is unnecessary unless we are going to draw a clear distinction